poc/common.sage

```diff
@@ -8,10 +8,10 @@ import struct
 
 # If set, then test vectors will be generated. A fixed source of randomness is
 # used for `gen_rand()`.
-TEST_VECTOR = False
+TEST_VECTOR = True
```

Comment:
Let's revert this, as `TEST_VECTOR == True` is only needed for generating test vectors.